contracts/modules/staking/StakingPool.sol

```diff
@@ -675,10 +684,15 @@ contract StakingPool is IStakingPool, ERC721 {
 
       rewards = premium * request.rewardRatio / REWARDS_DENOMINATOR;
       uint expireAtBucket = Math.divCeil(block.timestamp + request.period, BUCKET_DURATION);
-      uint _rewardPerSecond = rewards / (expireAtBucket * BUCKET_DURATION - block.timestamp);
+      // TODO UNDO the following change. Amplified the rewards given that this calculation was always 0 given that 
+      // rewards are too small compared to the denominator result
+      uint _rewardPerSecond = rewards * 10000 / (expireAtBucket * BUCKET_DURATION - block.timestamp);
 
       // 1 SLOAD + 1 SSTORE
       rewardBuckets[expireAtBucket].rewardPerSecondCut += _rewardPerSecond;
+
+      // TODO Temporary fix while debugging rewardPerSecond assignation
+      // rewardPerSecond += _rewardPerSecond;
```

Comment:
This was the temporary fix I mentioned I made to make the tests work, assigning rewards to `rewardPerSecond`